doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -5508,6 +5508,10 @@ Deactivate a Banner Message by its id (allows you to hide a message while retain
 
```

Comment:
I'll just add this comment here at the top...

Can you please add a release note snippet? https://guides.dataverse.org/en/6.3/developers/version-control.html#writing-release-note-snippets